packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -13,6 +13,7 @@ By default, the runtime is configured with a max batch size of `716800` bytes, w
 -   [Introduction](#introduction)
     -   [Compression](#compression)
     -   [Grouped batching](#grouped-batching)
+        -   [Risks](#risks)
```

Comment:
Nit:
```suggestion
    -   [Risks](#risks)
```

Reply:
`Risks` is a subsection to `Grouped batching`